Shared/Samples/Augment reality to collect data/AugmentRealityToCollectDataView.swift

```diff
@@ -0,0 +1,210 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct AugmentRealityToCollectDataView: View {
+    /// A scene with an imagery basemap.
+    @State private var scene: ArcGIS.Scene = {
+        // Creates an elevation source from Terrain3D REST service.
+        let elevationServiceURL = URL(
+            string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+        )!
+        let elevationSource = ArcGISTiledElevationSource(url: elevationServiceURL)
+        let surface = Surface()
+        surface.addElevationSource(elevationSource)
+        surface.backgroundGrid.isVisible = false
+        // Allow camera to go beneath the surface.
+        surface.navigationConstraint = .unconstrained
+        let scene = Scene(basemapStyle: .arcGISImagery)
+        scene.baseSurface = surface
+        scene.baseSurface.opacity = 0
+        return scene
+    }()
+    /// The AR tree survey service feature table.
+    private let featureTable = ServiceFeatureTable(
+        url: URL(string: "https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/rest/services/AR_Tree_Survey/FeatureServer/0")!
+    )
```

Comment:
This should be a state property so that it doesn't get re-created if the parent's body is re-computed.